fake_mail_server/tests/create_configs.rs

```diff
@@ -205,7 +205,7 @@ fn ok_greeting_codes() {
     ];
     for cap_combo in combos(&caps) {
         let mut cap_str = String::new();
-        cap_str.push_str("CAPABALITY");
+        cap_str.push_str("CAPABILITY");
```

Comment:
This one intrigues me

Reply:
This is (somewhat) old code that we originally wanted to use for generating test cases. This never really went in to use (but having this typo would have made for a fun debugging session 😄 )